R/rs_gee.R

```diff
@@ -0,0 +1,264 @@
+#' get_geomorph_landforms
```

Comment:
Are these all needed, did you write them all for this work or are they taken from somewhere?

Reply:
Not sure if they are needed- could be interesting to look at later.

I think I would recommend you that you skip them if you don't have a GEE account and `{rgee}` set up at the moment. Perhaps we can review them more carefully if we end up using those classifications in analysis?

I wrote these functions a while ago here: [surveyGEER](https://github.com/impact-initiatives-geospatial/surveyGEER)

Reply:
Ah okay, yeah it's just a lot of copying and pasting of complex functions. Are they all used? I would ast least remove ones that aren't

Reply:
gotcha - I removed the ones that aren't being used. The geomorphology and distance to closest water functions are currently in the targets pipeline so won't remove yet.